### PR TITLE
IBX-6692: `ibexa_get_anonymous_user_id` Twig helper added 

### PR DESCRIPTION
**Pull request by julitafalcondusza**
| Question      | Answer
| ------------- | ---
| JIRA Ticket   | (https://github.com/ibexa/storefront/pull/140)
| Versions      | 4.5, 4.6, master

`ibexa_get_anonymous_user_id` Twig helper added.

#### Checklist

- [x] Text renders correctly
- [x] Text has been checked with vale
- [x] Description metadata is up to date
- [x] Added link to this PR in relevant JIRA ticket or code PR

### CODE REVIEW

**Review by konradoboza — APPROVED** [review]

**Review by juskora — APPROVED** [review]

File: docs/templating/twig_function_reference/storefront_twig_functions.md [hunk 1]
```diff
@@ -102,4 +102,12 @@ You can use this, for example, to [parametrize the display of products by using
         })
     ) }}
 {% endif %}
+```
+
+### `ibexa_get_anonymous_user_id()`
+
+`ibexa_get_anonymous_user_id()` returns the configured UserID for the anonymous user.
```
Comment (mnocon) [inline docs/templating/twig_function_reference/storefront_twig_functions.md:109]:
```suggestion
`ibexa_get_anonymous_user_id()` returns the configured User ID for the anonymous user (configured in `ibexa.system.<scope>.anonymous_user_id`).
```

**Review by mnocon — APPROVED** [review]
I'm not sure about the placement of this snippet - it's not directly related to storefront and all the other functions in this file are prefixed with `ibexa_storefront_`.

On the other hand I don't see a better place for it now, so +1

**Comment by konradoboza** [comment]
@mnocon that was actually my suggestion - the best reasoning to me is that the code of this Twig helper actually resides in `ibexa/storefront` and that seems to be in-tact with our doc in general.

**Review by dabrt — APPROVED** [review]

**Comment by mnocon** [comment]
That's true, it comes from Storefront and I don't have a better suggestion where to place it, so it's ok.

Ideally I'd place it somewhere near `ibexa_user_get_current`, coming from `ibexa/user`. In my head these two are connectec/related, but "other twig filters" is not the place for them 😅

**Pull request merged**